concordium-std/src/impls.rs

```diff
@@ -1893,6 +1918,76 @@ fn parse_call_response_code(code: u64) -> CallContractResult<ExternCallResponse>
     }
 }
 
+#[inline(always)]
+fn is_invoke_failure_code(code: u64) -> bool {
+    code & 0xffff_ff00_0000_0000 == 0xffff_ff00_0000_0000
+}
```

Comment:
Please document a bit. This is hard to understand without having a lot of prior knowledge.

At least when this is inlined there is some comment, but when it is on its own it is har.d